include/tvm/runtime/c_runtime_api.h

```diff
@@ -82,6 +82,7 @@ typedef enum {
   kDLSDAccel = 6,
   kOpenGL = 11,
   // AddExtraTVMType which is not in DLPack here
```

Comment:
Move this comment after `kDLHexagon = 13`

Reply:
Will do.